EPPlus/EPPlus.MultiTarget.csproj

```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;netstandard2.0;net35;net40</TargetFrameworks>
-    <AssemblyVersion>4.5.3.2</AssemblyVersion>
-    <FileVersion>4.5.3.2</FileVersion>
-    <Version>4.5.3.2</Version>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp3.0;netstandard2.1;netcoreapp2.2;netcoreapp2.1;netstandard2.0;net40</TargetFrameworks>
```

Comment:
Why would you want to target netstandard2.1 and netstandard2.0? Wouldn't it just be better to keep the lower version?

Reply:
I was thinking about it, but ultimately decided to add several different targets because some projects I've seen or worked on do target them directly. For instance, I was unable to use any version of EPPlus on .Net Core 3.1 (the project required the use of .Net Core 3.1 because WPF and XAML Islands)

About netstandard2.1 and 2.0: Microsoft recommends that any new .net project should target either the latest .Net Core release (3.1) or the latest .Net Standard release (2.1), and discourages targeting older frameworks, unless necessary. But still, I've found issues trying to make EPPlus (both .Net Core 2.1 and .Net Standard 2.0) work on newer targets. So, to properly cover as much platforms as possible, I decided to directly target them in the .csproj file, without sacrificing existing projects that may depend on a specific framework version.

Reply:
Yeah, I think multi-targeting the various net core versions makes a lot of sense. I had just not considered targeting multiple net standard versions before.  But with netstandard likely going away with dotnet 5, this may be an unnecessary concern.